sdk/storage/azure-storage-file-datalake/src/test/java/com/azure/storage/file/datalake/UrlTests.groovy

```diff
@@ -0,0 +1,98 @@
+package com.azure.storage.file.datalake
+
+class UrlTests extends APISpec {
+
+    def "test custom url"() {
```

Comment:
Can you give a little more description of what's supposed to be happening here? Is this asserting that custom urls don't get modified for dfs/blob endpoints?

Reply:
yeah, essentially anything that isnt a .blob. or .dfs. type of URL should not get modified by the builders

Reply:
yes, I'll add some comments to each test